ios/RNRudderSdkModule.m

```diff
@@ -114,6 +114,36 @@ - (dispatch_queue_t)methodQueue
     [[RSClient sharedInstance] identify:_userId traits:_traits options:[self getRudderOptionsObject:_options]];
 }
 
+RCT_EXPORT_METHOD(alias:(NSString*)_newId options:(NSDictionary*)_options)
+{
+    if ([RSClient sharedInstance] == nil)
+    {
+        [RSLogger logWarn:@"Dropping the Alias call as RudderClient is not initialized yet, Please use `await` keyword with the setup call"];
+        return;
+    }
+    if([_newId isEqual:@""])
+    {
+        [[RSClient sharedInstance] alias:nil options:[self getRudderOptionsObject:_options]];
```

Comment:
We shouldn't support making alias with `newId` as nil/empty string, as the important property we need in an alias call is the `newId` which is supposed to replace the already `existingId`.

May be we can check if the `newId` is an empty string on the React Native module only and return if its invalid.

We should check the same for Android as well.